sale_commission/models/res_partner.py

```diff
@@ -22,6 +22,13 @@
 ##############################################################################
 from openerp import models, fields, api
 
+from .sale_commission import (
```

Comment:
Instead of this, it's easier to have directly a variable with the list of tuples:
```
settlement_periods = [
    ("monthly", "Monthly"),
    ("quaterly", "Quarterly"),
    ("semi", "Semi-annual"),
    ("annual", "Annual")]
```

There's no problem with the translations, as they are parsed the same way.